tests/integration/tests/api.rs

```diff
@@ -0,0 +1,390 @@
+//! API integration tests
```

Comment:
Nit: This file should be renamed to relayer_api due to content and future api tests for signers, notifications...